### PR TITLE
[PERFSCALE-3401] Add ceph step (BM CPT)

### PR DESCRIPTION
**Pull request by josecastillolema**
Out of scope for this iteration:
 - Use multus/secondary networks for the ceph network

### CODE REVIEW

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-x86-day2-odf

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-x86-day2-odf

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-x86-day2-odf

**Comment by josecastillolema** [comment]
/pj-rehearse pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-metal-x86-day2-odf

**Comment by josecastillolema** [comment]
```
# oc rsh -n openshift-storage $(oc get pods -n openshift-storage -o name -l app=rook-ceph-operator)
sh-5.1$ export CEPH_ARGS='-c /var/lib/rook/openshift-storage/openshift-storage.config'
sh-5.1$ ceph -s
  cluster:
    id:     fa448c3b-43af-42af-81bd-45ee0d5eff4b
    health: HEALTH_OK

  services:
    mon: 3 daemons, quorum a,b,c (age 11m)
    mgr: a(active, since 11m)
    mds: 1/1 daemons up, 1 hot standby
    osd: 24 osds: 24 up (since 10m), 24 in (since 11m)
    rgw: 1 daemon active (1 hosts, 1 zones)

  data:
    volumes: 1/1 healthy
    pools:   12 pools, 828 pgs
    objects: 342 objects, 132 MiB
    usage:   1.2 GiB used, 35 TiB / 35 TiB avail
    pgs:     0.121% pgs not active
             827 active+clean
             1   clean+premerge+peered
```

**Comment by josecastillolema** [comment]
cc @jeniferh

File: ci-operator/step-registry/openshift-qe/installer/bm/day2/odf/openshift-qe-installer-bm-day2-odf-commands.sh [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  # Setup socks proxy
+  sshpass -p "$(cat /bm/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@$bastion -fNT -D 12345
+  export KUBECONFIG=/tmp/kubeconfig
+  export https_proxy=socks5://localhost:12345
+  export http_proxy=socks5://localhost:12345
+  oc --kubeconfig=/tmp/kubeconfig config set-cluster bm --proxy-url=socks5://localhost:12345
+fi
+
+oc config view
+oc projects
```
Comment (rsevilla87) [inline ci-operator/step-registry/openshift-qe/installer/bm/day2/odf/openshift-qe-installer-bm-day2-odf-commands.sh:21]: Remove

Comment (josecastillolema) [reply ci-operator/step-registry/openshift-qe/installer/bm/day2/odf/openshift-qe-installer-bm-day2-odf-commands.sh:21]: Why? We seem to have this in all the steps, i.e.: https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh#L7-L8
Or do you mean to keep it only in performance related ones?

**Review by rsevilla87 — APPROVED** [review]
/lgtm

**Pull request merged**